src/main/java/org/prography/kagongsillok/review/infrastructure/ReviewRepositoryImpl.java

```diff
@@ -0,0 +1,30 @@
+package org.prography.kagongsillok.review.infrastructure;
+
+import static org.prography.kagongsillok.review.domain.QReview.review;
+
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.prography.kagongsillok.review.domain.Review;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class ReviewRepositoryImpl implements ReviewRepositoryCustom {
+
+    private static final int DEFAULT_SEARCH_RESULT_SIZE = 10;
+
+    private final JPAQueryFactory queryFactory;
+
+    @Override
+    public List<Review> findAllByMemberId(final Long memberId) {
+        return queryFactory
+                .selectFrom(review)
+                .where(
+                        review.memberId.eq(memberId),
+                        review.isDeleted.eq(false)
```

Comment:
오토 박싱 비용이 발생하는 코드네요 자바에서 Boolean 객체를 미리 두개 만들어두고 사용하게 해서 생성 비용을 아껴주고 있어요.
캐싱에 대해서 알아보시면 좋을 듯 합니다!
```suggestion
                        review.isDeleted.eq(Boolean.FALSE)
```